src/index.html

```diff
@@ -6,9 +6,33 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="./index.html">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="navigation">
+        <ul class="list">
+          <li class="list__li"><a class="is-active link" href="//">Apple</a></li>
```

Comment:
Let's fix href, code style and classes everywhere for the links, also add the empty line between the links
```suggestion
          <li class="nav__item">
            <a class="is-active nav__link" href="#apple">Apple</a>
          </li>
          
```

Reply:
Done